dietpi/dietpi-software

```diff
@@ -11615,6 +11655,70 @@ _EOF_
 			G_EXEC chown -R ffsync:ffsync /mnt/dietpi_userdata/firefox-sync
 
 		fi
+		
+                software_id=183 # Bitwarden RS
+                if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+                    Banner_Configuration
+
+                    Create_User -d /mnt/dietpi_userdata/bitwarden_rs bitwarden_rs
+
+                    # Make dir and data dir needed to run
+                    G_EXEC mkdir -p /mnt/dietpi_userdata/bitwarden_rs/data
+
+                    # Make and edit the template config to have no web vault
+		    if [[ ! -f '/mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env' ]]
+		    then
+			    G_EXEC cp /opt/bitwarden_rs/.env.template /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
+			    G_CONFIG_INJECT 'WEB_VAULT_ENABLED=' 'WEB_VAULT_ENABLED=false' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
```

Comment:
By default the working dir or users $HOME/data seems to be used as data dir, but would it make sense to set this explicitly to be failsafe?
```
G_CONFIG_INJECT 'DATA_FOLDER=' 'DATA_FOLDER=/mnt/dietpi_userdata/bitwarden_rs' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
```
That way the `data` sub dir could be skipped as well.

Reply:
I suppose that would make sense - I will add that now.